server/filestore.go

```diff
@@ -4800,6 +4830,13 @@ func (o *consumerFileStore) Update(state *ConsumerState) error {
 
 	// Replace our state.
 	o.mu.Lock()
+
+	// Check to see if this is an outdated update.
+	if state.Delivered.Consumer < o.state.Delivered.Consumer {
+		o.mu.Unlock()
+		return fmt.Errorf("old update ignored")
```

Comment:
Just checking: is an outdated really bad, indicative of a bug etc.. or is that "normal" in some situations? If normal, then we should return nil here otherwise upper-layer may fail.

Reply:
Was a bug, removed the bug but kept the check. So that does not get triggered now running full test suite.